packages/better-auth/src/adapters/kysely-adapter/kysely-adapter.ts

```diff
@@ -92,6 +92,14 @@ export const kyselyAdapter = (db: Kysely<any>, config?: KyselyAdapterConfig) =>
 				}
 				const { type = "sqlite" } = config || {};
 				let f = schema[model]?.fields[field];
+				if (!f) {
+					const foundField = Object.values(schema[model]?.fields || {}).find(
+						(f) => f.fieldName === field,
+					);
+					if (foundField) {
+						f = foundField;
+					}
+				}
```

Comment:
hey, this needs to be AFTER the next condition (`f = Object.values(schema).find((f) => f.modelName === model)!;`) otherwise it will still fail to find the field if the table name (model) was ALSO was renamed (ex: `verification` -> `auth_verification`)

you might want to add a test for that